File: Python/Estrutura condicional/baskara.py
# Biblioteca
import math

# Entrada de dados
a = float(input("Digite o coeficiente A: "))
b = float(input("Digite o coeficiente B: "))
c = float(input("Digite o coeficiente C: "))

# Algoritmo matemático
delta = math.pow(b, 2) - 4 * a * c

# Condicional
if delta < 0 or a == 0:
    print("Esta equação não possui raízes reais.")
else:
    x1 = (- b + math.sqrt(delta)) / (2 * a) # Precisa estar aqui em baixo, se não vai dar erro...
    x2 = (- b - math.sqrt(delta)) / (2 * a)
    print(f"O x1 é {x1:.4f} e o x2 é {x2:.4f}.")

File: C/Estrutura condicional/baskara.c
// Biblioteca(s)
#include <stdio.h>
#include <math.h>

int main()
{
    // Variáveis
    double a,b,c, delta, x1, x2;

    // Entrada de dados
    printf("Digite o coeficiente A: ");
    scanf("%lf", &a);
    printf("Digite o coeficiente B: ");
    scanf("%lf", &b);
    printf("Digite o coeficiente C: ");
    scanf("%lf", &c);


    // Algoritmo matemático
    delta = pow(b, 2) - 4 * a * c;

    // Condicional
    if (a == 0 || delta < 0) {
        printf("Esta equacao nao possui raizes reais.");
    }
    else {
        x1 = (-b+sqrt(delta))/(2*a);
        x2 = (-b-sqrt(delta))/(2*a);
        printf("O x1 é = %.4lf e o x2 é = %.4lf", x1, x2);
    }

    return 0;
}